scripts/install/plex.sh

```diff
@@ -51,15 +48,14 @@ echo "Updating system ... "
   usermod -a -G ${master} plex
 
 if [[ -n $claim ]]; then
-  sleep 5
```

Comment:
If a sleep is here, it's probably here for a reason.

Plex needs to be initialized and ready to receive requests in order for the server to be claimed. It is *very* likely my tests to claim were failing without a 5 second sleep here.